lms/static/js/learner_dashboard/views/entitlement_unenroll_view.js

```diff
@@ -0,0 +1,158 @@
+(function(define) {
+    'use strict';
+    define(['backbone',
+        'jquery',
+        'underscore',
+        'gettext',
+        'edx-ui-toolkit/js/utils/html-utils'
+    ],
+        function(Backbone, $, _, gettext, HtmlUtils) {
+            return Backbone.View.extend({
+                el: '#unenroll-entitlement-modal',
+
+                triggerSelector: '.js-action-unenroll-entitlement',
+
+                submitButtonSelector: '#unenroll-entitlement-submit',
+
+                trackInfoSelector: '#entitlement-track-info',
+
+                refundInfoSelector: '#entitlement-refund-info',
+
+                errorInfoSelector: '#unenroll-entitlement-error',
+
+                closeButtonSelector: '#unenroll-entitlement-modal .close-entitlement-modal',
+
+                mainPageSelector: '#dashboard-main',
+
+                initialize: function(options) {
+                    var view = this;
+                    this.dashboardUrl = options.dashboardUrl;
+
+                    this.$submitButton = $(this.submitButtonSelector);
+                    this.$trackInfo = $(this.trackInfoSelector);
+                    this.$refundInfo = $(this.refundInfoSelector);
+                    this.$errorInfo = $(this.errorInfoSelector);
+
+                    this.$submitButton.on('click', this.handleSubmit.bind(this));
+
+                    $(this.triggerSelector).each(function() {
+                        var $trigger = $(this);
+
+                        $trigger.on('click', view.handleTrigger.bind(view));
+
+                        if (window.accessible_modal) {
+                            window.accessible_modal(
+                                '#' + $trigger.attr('id'),
+                                view.closeButtonSelector,
+                                '#' + view.$el.attr('id'),
+                                view.mainPageSelector
+                            );
+                        }
+                    });
+                },
+
+                handleTrigger: function(event) {
+                    var $trigger = $(event.target),
+                        courseName = $trigger.data('courseName'),
+                        courseNumber = $trigger.data('courseNumber'),
+                        isRefundable = $trigger.data('entitlementIsRefundable'),
```

Comment:
We probably need to convert this to boolean.

`isRefundable = $trigger.data('entitlementIsRefundable') === 'True',`

Reply:
You can also do !!variable to check existance